schemas/Clinical/index.js

```diff
@@ -257,20 +258,15 @@ const convertClinicalSubmissionDataToGql = data => {
   const schemaErrors = get(data, 'schemaErrors', {});
   const fileErrors = get(data, 'fileErrors', []);
   // convert clinical entities for gql
-  const clinicalEntities = [];
-  for (let clinicalType in submission.clinicalEntities) {
-    clinicalEntities.push(
-      convertClinicalSubmissionEntityToGql(clinicalType, submission.clinicalEntities[clinicalType]),
-    );
-  }
-  // collect schema errors for each entity in dataErrors (not sure if this is OK??)
-  for (let clinicalType in schemaErrors) {
-    clinicalEntities.push(
-      convertClinicalSubmissionEntityToGql(clinicalType, {
-        dataErrors: schemaErrors[clinicalType],
-      }),
-    );
-  }
+  const clinicalEntities = Object.entries(submission.clinicalEntities).map(
```

Comment:
Hmm `Object.entries(submission.clinicalEntities)` can throw error. If submission doesn't exist in DB then default value is used from `const submission = get(data, 'submission', {});` where `submission.clinicalEntities` doesn't exist

Reply:
👍 nice catch, thanks!

Reply:
@jaserud address! :)

Reply:
:+1: